cmd/admin-heal-ops.go

```diff
@@ -528,19 +528,7 @@ func (h *healSequence) healDiskFormat() error {
 		return errServerNotInitialized
 	}
 
-	// Create a new set of storage instances to heal format.json.
-	bootstrapDisks, err := initStorageDisks(globalEndpoints)
-	if err != nil {
-		return errFnHealFromAPIErr(err)
-	}
-
-	// Wrap into retrying disks
-	retryingDisks := initRetryableStorageDisks(bootstrapDisks,
-		time.Millisecond, time.Millisecond*5,
-		globalStorageHealthCheckInterval, globalStorageRetryThreshold)
-
-	// Heal format.json on available storage.
-	hres, err := healFormatXL(retryingDisks, h.settings.DryRun)
+	hres, err := objectAPI.HealFormat(h.settings.DryRun)
```

Comment:
No need to add HealFormat() API which will come in in large bucket support.  you only need to hold a lock inside healDiskFormat on `format.json`

Reply:
Done.